src/data/move-attrs/copy-move-attr.ts

```diff
@@ -1,41 +1,40 @@
-import { type Pokemon, type PlayerPokemon, PokemonMove } from "#app/field/pokemon";
+import { allMoves } from "#app/data/all-moves";
+import { type Move } from "#app/data/move";
+import { CallMoveAttr } from "#app/data/move-attrs/call-move-attr";
+import { type MoveConditionFunc } from "#app/data/move-conditions";
+import { type Pokemon } from "#app/field/pokemon";
 import { globalScene } from "#app/global-scene";
-import { MovePhase } from "#app/phases/move-phase";
-import { type Move, getMoveTargets } from "#app/data/move";
-import { OverrideMoveEffectAttr } from "#app/data/move-attrs/override-move-effect-attr";
-import { lastMoveCopiableCondition, type MoveConditionFunc } from "../move-conditions";
+import type { BooleanHolder } from "#app/utils";
+import type { Moves } from "#enums/moves";
 
 /**
- * Attribute to copy the last move used in battle and invoke it against random target(s).
- * Used for {@link https://bulbapedia.bulbagarden.net/wiki/Mirror_Move_(move) | Mirror Move}
- * and {@link https://bulbapedia.bulbagarden.net/wiki/Copycat_(move) | Copycat}.
- * @extends OverrideMoveEffectAttr
+ * Attribute used to copy a previously-used move.
+ * Used for {@linkcode Moves.COPYCAT} and {@linkcode Moves.MIRROR_MOVE}
+ * @see {@linkcode apply} for move selection and move call
+ * @extends CallMoveAttr to call a selected move
```

Comment:
```suggestion
 * @extends CallMoveAttr
```
Current comment doesn't gel well with VSCode's doc formatter...